packages/rule-tester/package.json

```diff
@@ -51,6 +51,7 @@
     "@typescript-eslint/typescript-estree": "7.7.1",
     "@typescript-eslint/utils": "7.7.1",
     "ajv": "^6.12.6",
+    "json-stable-stringify-without-jsonify": "^1.0.1",
```

Comment:
This is the same package as used by ESLint since v4, so shouldn't cause any new deps for users who have `eslint` installed